.github/workflows/java-ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on: [push]
+
+jobs:
```

Comment:
Jeśli dobrze się orientuję to można wstawić CI do angulara. Pytanie czy pojawią się jakieś front-endowe testy? I czy jest to robota na teraz czy na inny sprint? Imo na razie nie trzeba, ale zostawiam komentarz jako sugestię.

Reply:
Myślę, że nie ma raczej większej potrzeby. Testów jednostkowych nie ma za bardzo jak zrobić na froncie. Integracyjnych też średnio. Nie jest to też za bardzo moja działka. Myślę, że można to póki co tak zostawić.